hdt-lib/src/hdt/BasicHDT.cpp

```diff
@@ -656,7 +657,7 @@ void BasicHDT::loadFromHDT(std::istream & input, ProgressListener *listener)
 	controlInformation.load(input);
 	std::string hdtFormat = controlInformation.getFormat();
 	if(hdtFormat!=HDTVocabulary::HDT_CONTAINER) {
```

Comment:
I think we're having an outdated discussion: https://github.com/rdfhdt/hdt-cpp/pull/36/files#diff-9b14bf8cdb754874c17bea746d616ae3R660